doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -430,6 +430,7 @@ system. The view can be accessed by Materialize superusers.
 <!-- RELATION_SPEC mz_internal.mz_notices -->
 | Field                   | Type                         | Meaning                                                                                                                                           |
 | ----------------------- | ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                    | [`text`]                     | Materialize's unique ID for this notice.                                                                                                          |
```

Comment:
How often does this table get cleared? We fill these in with transient IDs which are not unique across restarts.

Reply:
The table is cleared on every restart, so yes - the same conceptual notice will be re-generated on every restart with a different ID (unless something in the code changes that causes the notice condition to disappear).

The main motivation for that is that the UX currently needs some notion of key in the page that navigates to a notice. I was thinking that explicitly managing this key in `environmentd` is better than using some sort of hand-crafted hash, which the current console MVP PR suggests as follows:

```sql
md5(concat(
    n.message::text,
    '-',
    n.object_id::text
))
```